pkgs/applications/graphics/cyan/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qt5
+, cmake
+, pkg-config
+, imagemagick
+, nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cyan";
+  version = "1.2.4";
+
+  src = fetchFromGitHub {
+    owner = "rodlie";
+    repo = pname;
+    rev = version;
+    hash = "sha256-R5sj8AN7UT9OIeUPNrdTIUQvtEitXp1A32l/Z2qRS94=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [ imagemagick ];
+
+  passthru.updateScript = nix-update-script {
+    attrPath = pname;
+  };
+
+  meta = with lib; {
+    description = "Image viewer and converter, designed for prepress (print) work";
+    homepage = "https://github.com/rodlie/cyan";
+    license = licenses.cecill21;
+    mainProgram = "Cyan";
```

Comment:
Should we rename the binary to from `Cyan` to `cyan`? Then we can remove this line and `cyan` is easier to find and execute

Reply:
Way better. In these cases it is better to create a symlink.

Reply:
I don't think this is necessary for gui applications.